src/style.css

```diff
@@ -0,0 +1,111 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.containbig {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 70px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.containsmall {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 42px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.consearch {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  border-radius: 4px;
+  width: 100%;
+  height: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search--small {
+  color: #3d4e61;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  outline: none;
+}
```

Comment:
do not duplicate styles
check ones more

Reply:
made style edits